config/initializers/assets.rb

```diff
@@ -49,7 +48,8 @@
                   password_strength
                   proxy_status
                   about
-                  parameter_override)
+                  parameter_override
+                  charts)
```

Comment:
this file shouldnt really change, as there no changes (if you look at the diff, a line was moved over, which make no sense to include in the patch).